shanoir-ng-preclinical/src/main/resources/application.yml

```diff
@@ -73,10 +73,13 @@ spring:
 
 springdoc:
   api-docs:
-    path: /api-docs
+    path: '/api-docs'
+    enabled: true
   swagger-ui:
-    path: /swagger-ui.html
+    url: /shanoir-ng/studies/api-docs
+    config-url: /shanoir-ng/studies/api-docs/swagger-config

```

Comment:
shouldn't it be /shanoir-ng/preclinical instead of /shanoir-ng/studies here ?

Reply:
Hi @pierrehenri-dauvergne,

Should be OK now, thx.